### PR TITLE
Feature - Loaders 

### PR DESCRIPTION
**Pull request by ksookdhew**
## Description
Added loading indicators for project

## Checklist 
Please review and check the following before submitting this pull request: 
- [x] The code follows the project's coding standards. 
- [x] 100% Unit test coverage for View Models 
- [x] All warnings Resolved.

### CODE REVIEW

File: F1/F1/Race/RacingViewController.swift
Comment (AShozi) [inline F1/F1/Race/RacingViewController.swift:1]: Looks great!  make sure your consistent with how u declare your marks.
and the indenting on line 58 looks a little

**Review by AShozi — DISMISSED** [review]

File: F1/F1/Standings/StandingsViewController.swift [hunk 1]
```diff
@@ -7,17 +7,18 @@
...
     @IBOutlet weak private var tableView: UITableView!
-    @IBOutlet weak private var segControl: UISegmentedControl!
+    @IBOutlet weak private var segmentedControl: UISegmentedControl!
 
     // MARK: Variables
     var selectedSegmentIndex = 1
     private lazy var driverViewModel = DriverStandingsViewModel(repository: DriverStandingsRepository(), delegate: self)
     private lazy var constructorViewModel = ConstructorStandingsViewModel(repository: ConstructorStandingsRepository(), delegate: self)
     private lazy var standingsViewModel = StandingsViewModel(driverViewModel: driverViewModel, constructorViewModel: constructorViewModel, navigationDelegate: self)
+    private let dispatchGroup = DispatchGroup()
```
Comment (simekanidvt) [inline F1/F1/Standings/StandingsViewController.swift:21]: What is this used for?

Comment (ksookdhew) [reply F1/F1/Standings/StandingsViewController.swift:21]: Good catch, I was trying to use concurrency with the loading indicators to know when they should be dismissed. But decided to go a different route. Will remove it 😄

File: F1/F1/Race/RacingViewController.swift
Comment (ksookdhew) [reply F1/F1/Race/RacingViewController.swift:1]: Thanks will change!

**Review by AShozi — APPROVED** [review]

**Review by simekanidvt — APPROVED** [review]

**Review by kiasha10 — APPROVED** [review]

**Pull request merged**